### DISCUSSION TITLE
Illustrated message pool

### DISCUSSION THREAD

**Discussion by rnkab** [UI5/webcomponents#6667] [Q&A]
Hi,

I do see the names of all the illustrations supported OOTB here : https://sap.github.io/ui5-webcomponents/playground/components/IllustratedMessage/ 

However I was wondering if there is a repository where I can view how the illustrations look and their corresponding names (like the existing SAPUI5 icons repo) This one seems to be a bot outdated: https://pages.github.tools.sap/sap-design-ops/fiori-moments/all-illustrations.html  

Best,
Radhika

**Comment by pskelin** [comment]
You can use the test page here:
https://sap.github.io/ui5-webcomponents/playground/fiori/pages/IllustratedMessage/

But it certainly makes sense to have it in the docs page, I'll create a feature request.

**Reply by pskelin** [reply]
tracking #6670

**Reply by rnkab** [reply]
Thank you @pskelin . From the Digital Assistant Services(DAS) product perspective, we provide UI components to build a chat/conversation. These conversational components are based on the UI5 Web Components and for some of them we are planning to use the illustrated message underneath. So, ultimately a DA developer would have to provide us the name of the illustration which we pass down to UI5. So, can we also provide this link you shared above as a part of our documentation for our users? Or would you have a more stable link. (As of now our customers are SAP internal)
